docs/conf.py

```diff
@@ -71,11 +67,9 @@ def get_version():
 
 templates_path = ['_templates']
 
-master_doc = 'index'
-copyright = '2010-{}, Read the Docs, Inc & contributors'.format(
-    timezone.now().year
-)
-version = get_version()
+master_doc = "index"
+copyright = "2010-{}, Read the Docs, Inc & contributors".format(timezone.now().year)
```

Comment:
```suggestion
copyright = "2010, Read the Docs, Inc & contributors"
```
I think we can just do this for even less dynamic magic :)